config/initializers/decidim.rb

```diff
@@ -1,13 +1,11 @@
 # frozen_string_literal: true
 
 Decidim.configure do |config|
-  config.expire_session_after = ENV.fetch("DECIDIM_SESSION_TIMEOUT", 90).to_i.minutes
   config.application_name = "OSP Agora"
   config.mailer_sender = "OSP Agora <ne-pas-repondre@opensourcepolitics.eu>"
 
   # Change these lines to set your preferred locales
   config.default_locale = :en
-  config.available_locales = [:en, :fr]
```

Comment:
We shouldn't add all available locales here?

Reply:
I've deleted this line because it takes the default defined in Decidim-core, so we are sure to have all available locales for the given version !